client/tmclient/screens.py

```diff
@@ -335,3 +345,42 @@ def load_hotkeys(self):
                 }
 
         return hotkeys
+
+    def generate_minimap(self):
+        """Generates a minimap for the cardinal exits of the current room."""
+        room = self.state.get("room", {})
+        exits = room.get("exits", {})
+        blank = urwid.Text(" ")
+        map_nodes = {
+                "north": blank,
+                "east": blank,
+                "south": blank,
+                "west": blank,
+                "above": blank,
+                "below": blank,
+                }
+
+        for exit in exits.keys():
```

Comment:
it's annoying but I'd avoid using `exit` as a variable name since it's a built-in python function.

Reply:
my bad! just fixed it :)